website/templates/project/modal_add_contributor.mako

```diff
@@ -155,7 +155,13 @@
                                                 ><i class="fa fa-minus"></i></a>
                                         </td>
                                         <td>
-                                            <img class="m-v-xs" data-bind="attr: {src: contributor.gravatar_url}" width=35 height=35/>
+                                             <!-- height and width are explicitly specified for faster rendering -->
```

Comment:
@chennan47  -- NP: 5 spaces instead of 4 as outlined in the [style guide](http://cosdev.readthedocs.org/en/latest/style_guides/html_css.html#html-and-css).